code/index.html

```diff
@@ -2,21 +2,87 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooked News</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div></div>
+      <div></div>
+      <div></div>
+      <img src="https://thumbs.dreamstime.com/t/breaking-news-minimalistic-logo-white-background-entertaining-show-news-vector-illustration-breaking-news-headline-113066889.jpg" alt="news logo">
+      <nav>
+        <a href="#">Sports</a>
+        <a href="#">Politics</a>
+        <a href="#">USA</a>
+      </nav>
     </header>
 
     <section class="big-news">
-      <!-- The big news content -->
+      <section class="news">
+        <div> 
```

Comment:
you dont need to wrap this in a div as you already have a section with the class name news as the wrapper.